### PR TITLE
Add ajtrace repo

### PR DESCRIPTION
**Pull request by zilder**
Add [`ajtrace`](https://github.com/adjust/ajtrace) repo, a first minimal version of ebpf based tracing util for our production system.

### CODE REVIEW

File: dev-util/ajtrace/ajtrace-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+PYTHON_COMPAT=( python{3_4,3_5,3_6,3_7} )
+inherit distutils-r1 git-r3
+
+DESCRIPTION="eBPF based monitoring tool"
+HOMEPAGE="https://github.com/adjust/ajtrace"
+EGIT_REPO_URI="https://github.com/adjust/ajtrace.git"
+EGIT_BRANCH="master"
+
+LICENSE="MIT"
```
[the remark this answers is not in the document]
Comment (zilder) [reply dev-util/ajtrace/ajtrace-9999.ebuild]: done!

File: dev-util/ajtrace/Manifest [hunk 2]
```diff
@@ -0,0 +1 @@
+EBUILD ajtrace-9999.ebuild 493 BLAKE2B e42694703344a48656096ca4f6f26817f590855b6c56e7075fb42e16d8dc54b298c0ce7b96fb78d9b44e50540c076fba51dcef3bbfdea80ee3b6b7d402f819f2 SHA512 e37350f11801d5897256c63dec9ecb061e042b63b03a3d7b153cd250af0ed79cc55c601df3fbf5eff1a585d4cd23e75ab4b24ee58569316220757d01953f165f
```
[the remark this answers is not in the document]
Comment (zilder) [reply dev-util/ajtrace/Manifest]: won't `repoman` in travis script complain about it?

[the remark this answers is not in the document]
Comment (maxeaubrey) [reply dev-util/ajtrace/Manifest]: Nope, give it a try yourself :)

Or for a more precise answer, see repo config here https://github.com/adjust/gentoo-overlay/blob/master/metadata/layout.conf

**Comment by maxeaubrey** [comment]
```
# ajtrace 
Traceback (most recent call last):
  File "/usr/lib/python-exec/python3.6/ajtrace", line 11, in <module>
    load_entry_point('ajtrace==0.1.dev0', 'console_scripts', 'ajtrace')()
  File "/usr/lib64/python3.6/site-packages/ajtrace/ajtrace.py", line 45, in main
    config.initialize(args.config)
  File "/usr/lib64/python3.6/site-packages/ajtrace/settings.py", line 7, in initialize
    GlobalConfig.__config = yaml.load(open(path, 'r'))
  File "/usr/lib64/python3.6/site-packages/yaml/__init__.py", line 109, in load
    raise RuntimeError("Unsafe load() call disabled by Gentoo. See bug #659348")
RuntimeError: Unsafe load() call disabled by Gentoo. See bug #659348
```

**Comment by xmhd** [comment]
@zilder is this still desirable to have?

**Pull request closed without merging**